swagger.yml

```diff
@@ -19,7 +19,7 @@ responses:
   error:
     description: error
     schema:
-      $ref: "#/definitions/error"
+      $ref: '#/definitions/error'
```

Comment:
I'm not opposed to `s/"/'` but lets do it in a separate PR.  Can you unstage these quotation changes?

Reply:
We're using single quotes everywhere else, are you saying we should leave it as double quotes in this one location?

Reply:
Done

Reply:
I'm saying if we fix it, let's do it in a separate PR.

Reply:
Whitespace/formatting changes are best chunked into a single commit.

Reply:
Makes sense. Thanks. I fixed it so let me know if it's good to merge.